apps/root/src/state/index.ts

```diff
@@ -142,6 +143,7 @@ const createStore = (web3Service: Web3Service) =>
           theme: 'light',
           selectedLocale: SupportedLanguages.english,
           showSmallBalances: true,
+          showBalances: true,
```

Comment:
As falsy values are being removed from localStorage, I suggest we save `config` as an object rather than storing its values separately. Otherwise, we would be always having showBalances as true by default, and never saving user's preference

Reply:
Are you sure they are being removed? The case where you had that happen had a problem where you didn't add it to the preloaded states